cw-orch-daemon/src/senders/manual_sender.rs

```diff
@@ -0,0 +1,178 @@
+use crate::proto::injective::InjectiveEthAccount;
+use crate::queriers::Node;
+use crate::tx_broadcaster::assert_broadcast_code_cosm_response;
+use crate::{cosmos_modules, DaemonBase, TxBuilder};
+
+use crate::{error::DaemonError, tx_resp::CosmTxResponse};
+
+use cosmrs::proto::cosmos;
+use cosmrs::proto::cosmos::auth::v1beta1::BaseAccount;
+use cosmrs::proto::cosmos::vesting::v1beta1::PeriodicVestingAccount;
+use cosmrs::tendermint::chain::Id;
+use cosmrs::tx::{ModeInfo, Raw, SignDoc, SignMode, SignerInfo};
+use cosmrs::{AccountId, Any};
+use cosmwasm_std::Addr;
+use cw_orch_core::environment::ChainInfoOwned;
+use prost::Message;
+use tonic::transport::Channel;
+
+use std::io::{self, BufRead};
+
+use super::sender_trait::SenderTrait;
+
+pub type ManualDaemon = DaemonBase<ManualSender>;
+
+#[derive(Clone, Default)]
+pub struct ManualSenderOptions {
+    pub sender_address: Option<String>,
+}
+
+/// Signer of the transactions and helper for address derivation
+/// This is the main interface for simulating and signing transactions
+#[derive(Clone)]
+pub struct ManualSender {
```

Comment:
What's the purpose of manual sender?

Reply:
It's there to be able to broadcast a transaction using a multisig for instance, or when you want to send a transaction using other means. This is meant as an example to showcase the use of a Sender in this way, do you think it's worth keeping ?

Reply:
If it's an example, could you rename the file to `example_sender.rs` or similar?

Reply:
Or include it in `examples` folder instead of the bin?

Reply:
Done now